osu.Game.Rulesets.Rush/Judgements/CollisionDamagingJudgement.cs

```diff
@@ -11,11 +11,12 @@ namespace osu.Game.Rulesets.Rush.Judgements
     /// </summary>
     public class CollisionDamagingJudgement : RushJudgement
     {
-        protected override double HealthPointIncreaseFor(HitResult result, bool playerCollided) =>
-            result switch
-            {
-                HitResult.Miss when playerCollided => -10.0,
-                _ => 0.0,
-            };
+        protected override double HealthPointIncreaseFor(HitResult result, bool playerCollided)
```

Comment:
Would prefer expression syntax with a ternary for this one, since it's super short.